.github/workflows/example-chrome.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - uses: actions/upload-artifact@v3
         with:
           name: screenshots-in-electron
```

Comment:
Let's add the "-v9" to the name like the v10 version for consistency.

Reply:
I added -v9 to the test name and to the artifact names from v9.